apps/vr-tests-web-components/tsconfig.json

```diff
@@ -1,7 +1,8 @@
 {
   "extends": "../../tsconfig.base.wc.json",
   "compilerOptions": {
-    "target": "ES2019",
+    "target": "ES2022",
+    "useDefineForClassFields": false,
```

Comment:
can we move these to `tsconfig.base.wc.json` ?

Reply:
I updated the configs to remove the redundant/inherited fields.